wireshark_gen/templates/_ofclass_dissector.lua

```diff
@@ -43,7 +43,7 @@ function ${name}(reader, subtree)
 :: if ofclass.virtual:
     return ${ofclass.name}_v${version.wire_version}_dissectors[reader.peek(${ofclass.discriminator.offset},${ofclass.discriminator.length}):uint()](reader, subtree)
 :: else:
-:: if not ofclass.is_fixed_length:
+:: if not ofclass.is_fixed_length and ofclass.length_member is not None:
```

Comment:
Out of curiosity, what object is variable length and doesn't have a length member?